src/streaming/controllers/EventController.js

```diff
@@ -321,30 +323,49 @@ function EventController() {
     }
 
     /**
-     * Triggers an MPD reload
+     * Refreshes, patches, or updates the manifest based on the InbandEventStream value attribute.
+     * @param {object} event
+     * @private
+     */
+    function _handleManifestExiration(event, currentVideoTime) {
+        switch (event.eventStream.value) {
+            case MPD_RELOAD_VALUE:
+                logger.debug(`Starting manifest refresh event ${event.id} at ${currentVideoTime}`);
+                _refreshManifest();
+                break;
+            case MPD_PATCH_VALUE:
+                logger.debug(`Starting manifest patch event ${event.id} at ${currentVideoTime}`);
+                _patchManifest(event.messageData);
+                break;
+            case MPD_UPDATE_VALUE:
+                logger.debug(`Starting manifest patch update ${event.id} at ${currentVideoTime}`);
+                _updateManifest(event.messageData);
+                break;
+        }
+    }
+    /**
+     * Triggers an MPD reload.
      * @param {object} event
      * @private
      */
     function _handleManifestReloadEvent(event) {
```

Comment:
I think that we can change the name of this method to _handleManifestValidityExpirationEvent
```suggestion
    function _handleManifestValidityExpirationEvent(event) {
```